mobile/src/main/java/ch/epfl/sdp/mobile/ui/prepare_game/PrepareGameScreen.kt

```diff
@@ -0,0 +1,64 @@
+package ch.epfl.sdp.mobile.ui.prepare_game
+
+import androidx.compose.foundation.layout.*
+import androidx.compose.material.MaterialTheme
+import androidx.compose.material.Text
+import androidx.compose.runtime.*
+import androidx.compose.ui.Alignment
+import androidx.compose.ui.Modifier
+import androidx.compose.ui.tooling.preview.Preview
+import androidx.compose.ui.unit.dp
+import ch.epfl.sdp.mobile.state.LocalLocalizedStrings
+import ch.epfl.sdp.mobile.ui.PawniesTheme
+
+@Preview(showBackground = true, backgroundColor = 4294966246)
+@Composable
+fun preview_home() {
+  val state =
+      object : PrepareGameScreenState {
+        override var colorChoice: ColorChoice = ColorChoice.WHITE
+        override var gameType: GameType = GameType.LOCAL
+        override val onNewLocalGame: () -> Unit = {}
+        override val onNewOnlineGame: () -> Unit = {}
+      }
+  PawniesTheme { PrepareGameScreen(state, Modifier.padding(16.dp, 16.dp)) }
+}
+
+@Composable
+fun PrepareGameScreen(
+    state: PrepareGameScreenState,
+    modifier: Modifier = Modifier,
+    paddingValues: PaddingValues = PaddingValues()
+) {
+  val strings = LocalLocalizedStrings.current
+  var colorChoice by remember { mutableStateOf(state.colorChoice) }
+  var gameType by remember { mutableStateOf(state.gameType) }
```

Comment:
You should not keep state here, but hoist it in the (already provided) `state` argument.